client/widgets/db-stats/stats.template.html.slim

```diff
@@ -0,0 +1,56 @@
+.db-stats.panel.panel-default
+  .panel-heading
+    help-button
+      p Help will be there
+    h3.panel-title DB table metrics
+  .panel-body
+    .row ng-if='loading && stats.length === 0'
```

Comment:
This is never displayed, as `$scope.stats` is only set once the request is complete, therefore `$scope.stats.length` is undefined.